Makefile

```diff
@@ -163,12 +163,11 @@ update-contextual-logging: $(LOGCHECK) ## Update contextual logging
 .PHONY: generate-cli-docs
 generate-cli-docs: ## Generate cli docs
 	git clean -fdX docs/content/reference/cli
-	pushd . && cd docs/generators/cli-doc && go build . && popd
-	./docs/generators/cli-doc/cli-doc -output docs/content/reference/cli
+	cd ./docs/generators/cli-doc && go run . -output ../../content/reference/cli
```

Comment:
Why change this? It was working perfectly fine and now it's changing directories 🤷🏻